style.css

```diff
@@ -0,0 +1,115 @@
+
+body {
+    text-align: center;
+    width: 100%;
+    line-height: 3em;
+}
+
+#keyboard {
+    display: flex;
```

Comment:
Usually we used Ids for JS, for css it's not good way to use ids like selectors.

Reply:
Okey, i agree

Reply:
I've just corrected it